spec/unit/shared/property_spec.rb

```diff
@@ -176,4 +176,52 @@ def to_ruby(value)
       expect(instance.class.declared_properties.convert_properties_to(instance, :ruby, instance.props)[:range]).to eq(range)
     end
   end
+
+  describe Neo4j::ActiveNode do
+    before(:each) do
+      # This serializer adds a text when the data is saved on the db,
+      # and removes it when deserializing
+      class MySerializer
```

Comment:
I would use `stub_named_class`.  When we define classes directly in the specs we end up with weird issues.